docs/operations/cx-server-lifecycle.md

```diff
@@ -0,0 +1,209 @@
+## Operations Guide for Cx Server
```

Comment:
Why change the name of this file? I find 'operations guide' to be more descriptive.

Reply:
I expect there will be other operations related files. Which then can be added to this directory.